contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -982,6 +982,91 @@ $BODY$
 LANGUAGE plpgsql
 IMMUTABLE;
 
+create or replace function sys.babelfish_timezone_mapping(IN tmz text) returns text
+AS 'babelfishpg_tsql', 'timezone_mapping'
+LANGUAGE C IMMUTABLE ;
+
+CREATE OR REPLACE FUNCTION sys.timezone(IN tzzone PG_CATALOG.TEXT ,  IN input_expr PG_CATALOG.TEXT)
+RETURNS sys.datetimeoffset
+AS
+$BODY$
+BEGIN
+    IF input_expr = 'NULL' THEN
```

Comment:
will last condition not cover this as well, why extra check for 'NULL'

Reply:
Need to cover this case : -
```
 Select 'NULL' AT TIME ZONE NULL
 GO
```